modules/migration-mtc-workflow.adoc

```diff
@@ -7,7 +7,19 @@
 [id='migration-mtc-workflow_{context}']
 = {mtc-full} workflow
 
-Migrating an application with the {mtc-short} web console involves the following workflow:
+The {mtc-full} ({mtc-short}) enables you to migrate Kubernetes resources, persistent volume data, and internal container images from an {product-title} source cluster to an {product-title} {product-version} target cluster, using the {mtc-short} web console or the Kubernetes API.
```

Comment:
```suggestion
You use the {mtc-full} ({mtc-short}) to migrate Kubernetes resources, persistent volume data, and internal container images from an {product-title} source cluster to an {product-title} {product-version} target cluster by using the {mtc-short} web console or the Kubernetes API.
```

Reply:
@apinnick, what do you think of the suggested edit?

Reply:
Fine